src/Altinn.App.Core/Models/Logo.cs

```diff
@@ -0,0 +1,25 @@
+using Altinn.Platform.Storage.Interface.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Altinn.App.Core.Models
+{
+    /// <summary>
+    /// The Logo configuration
+    /// </summary>
+    public class Logo
+    {
+        /// <summary>
+        /// A flag to specify that the form should display appOwner in header
+        /// </summary>
+        [JsonProperty(PropertyName = "showAppOwnerInHeader")]
+        public bool ShowAppOwnerInHeader { get; set; }
```

Comment:
The config field `ShowAppOwnerInHeader` does not make it clear to me what is show/hidden. Is it possible to find a better name for this config field?

Reply:
Hmm. What we actually are showing is the AppOwnerName. Would `DisplayAppOwnerNameInHeader` be more clear? Or do you have any suggestions? 
I am having a hard time coming up with anything else :sweat_smile:

Reply:
`ShowAppOwnerNameInHeader` or `DisplayAppOwnerNameInHeader` would both work for me.
Imo this makes it clear that it's connected to the name not the logo in the header

Reply:
Nice. I will go for `DisplayAppOwnerNameInHeader`